kernel/ramfs.c

```diff
@@ -2326,6 +2326,15 @@ static int _fs_ioctl(
             ret = devfs_get_pts_id(file, id);
             break;
         }
+        case TCGETS:
+        case TCSETS:
+        case TCSETS + 1:
```

Comment:
any reason why not using TCSETSW / TCSETF directly?

Reply:
I don't think those are defined in MUSL.